src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -82,7 +99,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
 };
 
 const Small: React.FC<ChannelImageBadgeProps> = ({type}) => {
-  if (type === 'COMMUNITY') {
+  if (type === 'TOPIC') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, changing the check from `type === 'COMMUNITY'` to `type === 'TOPIC'` could potentially cause issues. Ensure all instances of this component usage have been updated.
```diff
-  if (type === 'COMMUNITY') {
+  if (type === 'TOPIC') {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->